### DISCUSSION TITLE
Skill calc feature

### DISCUSSION THREAD

**Discussion by OrenWiesz** [runelite/runelite#16580] [Ideas]
maybe unintentional, but for the past 2 years (at least) the skill calc used to update the current xp to your current xp when you clicked on a skill icon. ie, you go to cooking skill. it automatically shows youre at 11,131,232 exp as that is your current exp. you change the current exp to level 1 to check for example how many sharks it would take to reach from level 1 to 99, and then if you click on the cooking icon again, it would update your current exp to 11,131,232 (assuming its still your current exp)

recently the plugin lost said functionality, and i was told it was never intended, so im suggesting to make it intended as its a really neat and useful feature :)

![image](https://github.com/runelite/runelite/assets/133283363/eaf9f844-75b4-427b-a3d0-7c33b890cd38)

**Comment by Nightfirecat** [comment]
It looks like that behavior caused a pretty severe bug in other places where it was used--primarily in the configuration panel when clicking the plugin hub tab when it was already open. It was changed here: https://github.com/runelite/runelite/commit/5d236f6379236975eb70a452437b6cfe278b7b80

We could add the ability to make only certain panels able to re-open like that, but I also don't see this as a highly necessary feature.
